models/user.test.js

```diff
@@ -0,0 +1,233 @@
+'use strict';
+
+const { NotFoundError, BadRequestError, UnauthorizedError } = require('../expressError');
+const db = require('../db.js');
+const User = require('./user.js');
+const { commonBeforeAll, commonBeforeEach, commonAfterEach, commonAfterAll } = require('./_testCommon');
+
+beforeAll(commonBeforeAll);
+beforeEach(commonBeforeEach);
+afterEach(commonAfterEach);
+afterAll(commonAfterAll);
+
+/************************************** authenticate */
```

Comment:
Similar to some of my previous comments, the code already describes what it does hence you don't need the comment

Reply:
I removed all unnecessary comments.